src/goto-harness/recursive_initialization.cpp

```diff
@@ -1008,6 +1009,27 @@ code_blockt recursive_initializationt::build_function_pointer_constructor(
   for(const auto &target : targets)
   {
     auto const assign = code_assignt{dereference_exprt{result}, target};
+    auto const sym_to_lookup =
```

Comment:
Aside comment:

FWIW I think we should _probably_  do something cleverer for function pointers, this is very similar to function pointer removal.